prov/verbs/src/verbs_domain.c

```diff
@@ -494,6 +532,14 @@ fi_ibv_domain(struct fid_fabric *fabric, struct fi_info *info,
 		goto err3;
 	}
 
+	if (!strncmp(info->domain_attr->name, "hfi1", strlen("hfi1"))) {
```

Comment:
Do we need to add checks for the previous generations (before OPA)?  I don't recall what those were called though.

Reply:
I think it was `qib` but I'm not sure what's the behavior of its verbs implementation.

Reply:
In terms of completions qib shares the same rdmavt completion logic.

The issue should be there as well.

Reply:
> In terms of completions qib shares the same rdmavt completion logic.
The issue should be there as well.

I'll do a follow-up patch add qib.